### DISCUSSION TITLE
关于长时间播放视频，帧率突然下降，显卡占用骤降

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#370] [mpv 通用]
笔记本看番，加载了几个GPU占用在50-60%的补祯滤镜，成功加载，但是看几分钟就会掉帧，gpu占用率骤降到个位数，笔记本调整到性能模式也不行。以为是散热压力过大，但调整到占用30左右的滤镜仍会出现以上情况，大概是哪些问题啊？

**Comment by hooke007** [comment]
从没遇到过，只能猜测是供电问题

**Reply by sqq135** [reply]
怀疑是长时间没有操作自动省电，但是我是插着电源的啊。这大概无关技术，只是需要个小技巧
